core/domain/draft_upgrade_services.py

```diff
@@ -88,6 +90,135 @@ def try_upgrading_draft_to_exp_version(
 class DraftUpgradeUtil(python_utils.OBJECT):
     """Wrapper class that contains util functions to upgrade drafts."""
 
+    @classmethod
+    def _convert_states_v33_dict_to_v34_dict(cls, draft_change_list):
+        """Converts draft change list from state version 33 to 34. State
+        version 34 adds the new schema for Math RTEs.
+
+        Args:
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+        """
+        for i, change in enumerate(draft_change_list):
+            if (change.cmd == exp_domain.CMD_EDIT_STATE_PROPERTY and
+                    change.property_name ==
+                    exp_domain.STATE_PROPERTY_INTERACTION_ANSWER_GROUPS):
+                converted_new_value = (
+                    [(
+                        state_domain.AnswerGroup.convert_html_in_answer_groups(
+                            answer_group,
+                            html_validation_service.
+                            add_math_content_to_math_rte_components))
```

Comment:
Create a `conversion_function = html_validation_service.add_math_content_to_math_rte_components` variable on top and use it.

Reply:
done